test/GTKT/burnable_tests.js

```diff
@@ -0,0 +1,102 @@
+let GTKT = artifacts.require("StandardMintableToken");
+
+const assertFail = require("../helpers/assertFail");
+
+let gtkt;
+
+contract("GTKT", function(accounts) {
+  beforeEach(async () => {
+
+    frozen120addr = accounts[4];
+    frozen365addr = accounts[5];
+    gtkt = await GTKT.new(
+        "Gold Ticket",               // the token name
+        8,             // amount of decimal places in the token
+        "GTKT",             // the token symbol
+        100000000000000000000           // the initial distro amount
+        );
+
+    await gtkt.transfer(accounts[1], 50000, {
+      from: accounts[0]
+    });
+
+    await gtkt.transfer(accounts[2], 50000, {
+      from: accounts[0]
+    });
+
+  });
+
+  /*
+  * Burn you own
+  */
+  it("User can burn their own tokens", async () => {
+    await gtkt.burn(50000, {
+      from: accounts[0]
+    });
+    assert.equal((await gtkt.balanceOf.call(accounts[0])).toNumber(), 99999999999999850000);
+  });
+
+  it("User shouldn't be able to burn more tokens than they have", async () => {
+    await assertFail(async () => {
+      await gtkt.burn(100000000000000000001, {
```

Comment:
careful, this number might be too big for javascript to handle properly. Perhaps use BigNumber in these cases